### DISCUSSION TITLE
Setting when triggers trigger.

### DISCUSSION THREAD

**Discussion by dtkaplan** [qmd-lab/closeread#86] [General]
In the Menard's map example in the gallery, you've arranged things so that a trigger doesn't take effect until it is half-way up the narrative column. This behavior is defined somewhere in the tufte.css file, but I can't figure out where that is. 

It would be helpful if the documentation indicated how to achieve this "delayed triggering," without imposing all the other features of tufte.css.

**Comment by andrewpbray** [comment]
Agreed! That effect is created in a rather inelegant way that's described in #84 . I've added an issue to that we can keep track of this #88.
